pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/Functions.java

```diff
@@ -19,10 +19,17 @@
 package org.apache.pulsar.broker.admin.v1;
 
 import io.swagger.annotations.Api;
+
+import javax.ws.rs.Consumes;
 import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
 import org.apache.pulsar.broker.admin.impl.FunctionsBase;
 
 @Path("/functions")
 @Api(value = "/functions", description = "Functions admin apis", tags = "functions", hidden = true)
-public class Functions extends FunctionsBase {
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class Functions extends  org.apache.pulsar.broker.admin.v2.Functions{
```

Comment:
need to do this because the 2.2 admin client hits the v1 endpoints i.e. /admin/functions